### ISSUE TITLE
SDRplay not recognized after update 28-1-2024

### ISSUE THREAD

**Issue by loewal** [AlexandreRouma/SDRPlusPlus#1300]
**Hardware**
- CPU: Mac Intel
- RAM: 8Gb
- GPU: Intel Core Duo i5
- SDR: SDRplay RSP1

**Software**
- MacOs Monterey 12.7.2
- SDR++: 1.1.0 built 27-1-2024

**Bug Description**
The SDRplay module is not present anymore

**Steps To Reproduce**
1. started earlier version; SDRplay was shown
2. installed named version
3. no more SDRplay selectable

**Additional info**
Luckily I could restore a version from dec 2023. SDRplay is there again!!

**Comment by AlexandreRouma** [comment]
Update the SDRplay API to 3.12.1.

**Comment by delboy85341** [comment]
[AlexandreRouma] commented [last week]

Update the SDRplay API to 3.12.1.

@AlexandreRouma I just went to https://www.sdrplay.com/api/ to download the API.  The version that's there now is 3.14.  I installed it but SDR++ doesn't see my RSP1A.  Thanks for all the work you put into SDR++.  I used to work in IT and had to troubleshoot programs when they didn't work as expected.  I know what a pain in the  backside it can be to track down somehting.
